ci/pipelines/builder.yml

```
@@ -1193,3 +1211,4 @@ resources:
   source:
     regexp: azure-storage-cli-(.*)-linux-amd64
     bucket: bosh-azure-storage-cli-artifacts
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)